resources/datomic/schema.edn

```diff
@@ -1,16 +1,48 @@
 [;; User entities
  {:db/id          #db/id [:db.part/db]
-  :db/ident       :user/username
+  :db/ident       :user/id
+  :db/valueType   :db.type/string
+  :db/unique      :db.unique/identity
+  :db/cardinality :db.cardinality/one
+  :db/doc         "A user's public id"}
+
+ {:db/id          #db/id [:db.part/db]
+  :db/ident       :user/handle
   :db/valueType   :db.type/string
   :db/cardinality :db.cardinality/one
-  :db/doc         "A user's username"}
+  :db/doc         "A user's wombat handle"}
 
  {:db/id          #db/id [:db.part/db]
-  :db/ident       :user/github-name
+  :db/ident       :user/access-token
   :db/valueType   :db.type/string
   :db/unique      :db.unique/identity
   :db/cardinality :db.cardinality/one
-  :db/doc         "Users GitHub username."}
+  :db/doc         "A user's GitHub auth token."}
+
+ {:db/id          #db/id [:db.part/db]
+  :db/ident       :user/github-username
+  :db/valueType   :db.type/string
+  :db/cardinality :db.cardinality/one
+  :db/doc         "A user's GitHub username."}
+
+ {:db/id          #db/id [:db.part/db]
+  :db/ident       :user/github-id
+  :db/valueType   :db.type/long
+  :db/cardinality :db.cardinality/one
+  :db/doc         "A user's GitHub id."}
+
+ {:db/id          #db/id [:db.part/db]
+  :db/ident       :user/email
+  :db/valueType   :db.type/string
+  :db/unique      :db.unique/identity
+  :db/cardinality :db.cardinality/one
+  :db/doc         "A user's email from their GitHub account."}
+
+ {:db/id          #db/id [:db.part/db]
+  :db/ident       :user/avatar-url
+  :db/valueType   :db.type/string
+  :db/cardinality :db.cardinality/one
+  :db/doc         "A user's avatar url from their GitHub account."}
```

Comment:
Why are we storing the user's avatar url? Couldn't we just rely on the Github API to retrieve this? Seems like it could be an unneeded pain. We'd have to ensure that the url is in sync with the account.